### PR TITLE
Michelle Cali

### PR DESCRIPTION
**Pull request by michellecali**

### CODE REVIEW

File: test_3_add_up_numbers.py [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 def add_up_numbers(arr):
   # Remove the `return None` code below and add your own code.
-  return None
-
+ return sum(int(x) for x in arr)
```
Comment (halfghaninne) [inline test_3_add_up_numbers.py:5]: Can you tell me more about the expression `int(x) for x in arr` and your thought process for including it?

File: test_4_filter_numbers.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 def filter_numbers(arr):
   # Remove the `return None` code below and add your own code.
-  return None
+  return [a for a in arr if a < 10]
```
Comment (halfghaninne) [inline test_4_filter_numbers.py:3]: 💪  Nice.

File: test_3_add_up_numbers.py (hunk 1 above)
Comment (michellecali) [inline test_3_add_up_numbers.py:5]: Somewhere out on the interwebs, I read that a best practice for use of the python sum function when using "raw input" was to convert it to a number to ensure it works. 

It was either this discussion, or something similar: 
http://stackoverflow.com/questions/11344827/summing-elements-in-a-list

Thx!

Comment (halfghaninne) [inline test_3_add_up_numbers.py:5]: Nice 👍

**Comment by halfghaninne** [comment]
Good work 👍

**Pull request closed without merging**